spec/models/miq_report_result/purging_spec.rb

```diff
@@ -111,15 +111,18 @@
       end
     end
 
+    # private method - not sure if we need
     it "#purge_ids_for_remaining" do
       described_class.send(:purge_ids_for_remaining, 1).should == {1 => @rr1.last.id, 2 => @rr2.last.id}
     end
 
+    # private method - not sure if we need (and this is very expensive)
     it "#purge_counts_for_remaining" do
       described_class.send(:purge_counts_for_remaining, 1).should == {1 => 1, 2 => 2}
     end
 
     context "#purge_count" do
+      # private method - not sure who uses (and this is very expensive)
```

Comment:
Used by the purge scripts as mentioned above.  It would not be called in automated purging via schedule.